README.md

```diff
@@ -1,2 +1,146 @@
-# leaderboard
-A leaderboard connected to an API to display the top scores of a game.
+<div align="center">
+
+  <h1><b>Leaderboard</b></h1>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
```

Comment:
- Hey, since you've removed the deployment from the readme, please, also remove it from the readme navigation to ensure users do not click links that are broken to make your readme more professional.

Reply:
Updated, thanks!